specifyweb/frontend/js_src/lib/hooks/__tests__/useBooleanState.test.ts

```diff
@@ -0,0 +1,167 @@
+import { act, renderHook } from '@testing-library/react';
+import { useBooleanState } from '../useBooleanState';
+
+test('Verify state does not change when setter not called with default value', () => {
+  const { result } = renderHook(() => useBooleanState());
+
+  expect(result.current[0]).toBe(false);
+});
+
+test('Verify state does not change when setter is not called', () => {
+  const { result } = renderHook(() => useBooleanState(true));
+
+  expect(result.current[0]).toBe(true);
+});
+
+test.each([
```

Comment:
I had no idea this was allowed!
that's so cool - reduces the need for `theories()`